sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -118,6 +121,10 @@ class Explorer extends Foundation<
     private windowResizing: number;
     private editor: monaco.editor.IStandaloneCodeEditor;
     private editorContainerRef: React.RefObject<HTMLDivElement> = React.createRef();
+    private adaptor: MonacoAdaptor;
```

Comment:
"adapter"

Reply:
Well, I am going to have to do a scrub after this haha, good thing I didn't export it from root.

Reply:
Issue filed to fix this - https://github.com/microsoft/fast/issues/3961

At this time the MonacoAdaptor has not been exported from root or documented so this is a safe non-breaking change.